deploy/distrib/src/main/resources/jboss/bin/standalone.conf

```diff
@@ -71,6 +71,6 @@ fi
 
 # Set some Bonita variables
 PLATFORM_SETUP=`cd $DIRNAME/../setup; pwd`
-INCIDENT_LOG_DIR=`cd $DIRNAME/../standalone/log; pwd`
-BONITA_OPTS="-Dorg.bonitasoft.platform.setup.folder='${PLATFORM_SETUP}' -Dorg.bonitasoft.engine.incident.folder='${INCIDENT_LOG_DIR}'"
+INCIDENT_LOG_DIR=`cd $DIRNAME/../standalone; pwd`
+BONITA_OPTS="-Dorg.bonitasoft.platform.setup.folder='${PLATFORM_SETUP}' -Dorg.bonitasoft.engine.incident.folder='${INCIDENT_LOG_DIR}/log'"
```

Comment:
why didn't you keep INCIDENT_LOG_DIR=`cd $DIRNAME/../standalone/log; pwd` and may be add mkdir if needed ?